packages/bot/src/components/run-panel.jsx

```diff
@@ -46,19 +49,57 @@ const drawerFooter = (props) => {
     );
 };
 
-const RunPanel = () => {
-    const content = drawerContent();
-    const footer = drawerFooter(onRunBotClick);
+// Temperarily change to state component to test animation
+class RunPanel extends React.PureComponent {
+    constructor() {
+        super();
+        this.state = {
+            contract_status: 0,
+        };
+    }
 
-    return (
-        <Drawer
-            className='run-panel'
-            is_open={true}
-            footer={footer}
-        >
-            {content}
-        </Drawer>
-    );
-};
+    componentDidMount () {
+        setInterval(() => {
```

Comment:
@sam-binary handle this in store. we're using mobx for state management, we dont need to use react states.

Reply:
this is just temporarily, so show the animation. Once integrated with Aaron part, all these will be removed